code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -647,23 +671,48 @@
 	if(ishuman(M))
 		var/mob/living/carbon/human/H = M
 		H.add_chemical_effect(CE_CLUMSY, 1)
+
 		for(var/obj/item/organ/internal/I in H.internal_organs)
 			if(I.organ_tag == BP_BRAIN)
 				if(I.damage >= I.min_bruised_damage)
 					continue
 			if((I.damage > 0) && (I.robotic != 2)) //Peridaxon heals only non-robotic organs
-				I.damage = max(I.damage - removed, 0)
+				var/damage_healed = ((M.bodytemperature < 186) && M.chem_effects[CE_CRYO]) ? 2 : 1 //2x effective if administered in cryogenic conditions
+				I.damage = max(I.damage - damage_healed*removed, 0)
+
+		if((M.bodytemperature < 153) && M.chem_effects[CE_CRYO] && M.chem_effects[CE_ANTIBIOTIC]) //peridaxon in extracool cryogenic conditions alongside antibiotics will have a chance to de-nercotise liver and kidneys, though will incur overdose symptoms
+			overdose(M, alien, removed, holder)
```

Comment:
this also doing the dizziness effects seems a little silly, I'd probably just split the tumour part into another proc on peridaxon and then only call that here